README.md

```diff
@@ -337,6 +345,32 @@ This section describes all possible config options.
 
 #### **Add Config**
 
+|option|description|type|default|
+|---|---|---|---|
+|chunker|chunker config|ChunkerConfig|{"chunk_size": 4000, "chunk_overlap": 200, "length_function": len}|
```

Comment:
default value is category specici here.

Reply:
I'll just mention to refer the table below for default values of different data_types